workspaces/ui-v2/src/entry-points/cloud/TopLevelRoutes.tsx

```diff
@@ -11,7 +11,7 @@ export default function TopLevelRoutes() {
         <Route
           strict
           //@TODO: centralize this path pattern
-          path="/public-specs/:specId"
+          path="/public-specs/:personId/:specId"
```

Comment:
Would it make sense to change the resource hierarchy so it's a person who has public-specs, as opposed to the other way around? How would you accommodate an org having public specs as well?